ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java

```diff
@@ -16,7 +16,6 @@ public LdesMemberIngestionController(final FragmentationService ldesReader) {
         this.ldesReader = ldesReader;
     }
 
-    // TODO update value to an injected value
     @PostMapping(value = "/ldes-member", consumes = "application/n-quads")
```

Comment:
The main point for this TODO was to maybe replace this with /{shortname} . Is this something that we might want?